components/tools/OmeroPy/src/omero/testlib/__init__.py

```diff
@@ -296,49 +298,17 @@ def import_image(self, filename=None, client=None, extra_args=None,
                     pass
         return pix_ids
 
-    """
-    Creates a fake file with one image, imports
-    the file and then return the image.
-    """
-
-    def import_single_image(self, name=None, client=None,
-                            with_companion=False, **kwargs):
-        if client is None:
-            client = self.client
-        if name is None:
-            name = "import_single_image"
-
-        images = self.import_mif(1, name=name, client=client,
-                                 with_companion=with_companion,
-                                 **kwargs)
-        return images[0]
-
-    """
-    Creates a fake file with one image and a companion file, imports
-    the file and then return the image..
-    """
-
-    def import_single_image_with_companion(self, name=None, client=None):
-        if client is None:
-            client = self.client
-        if name is None:
-            name = "import_single_image_with_companion"
-
-        images = self.import_mif(1, name=name, client=client,
-                                 with_companion=True)
-        return images[0]
-
-    """
-    Creates a fake file with a seriesCount of images, imports
-    the file and then return the list of images.
-    """
-
-    def import_mif(self, series_count=0, name=None, client=None,
-                   with_companion=False, skip="all", **kwargs):
+    def import_fake_file(self, images_count=1, name=None, client=None,
+                         with_companion=False, skip="all", **kwargs):
+        """
+        Creates a fake file with a seriesCount of images, imports
```

Comment:
`s/seriesCount/images_count`. Otherwise, the name of the new method makes sense and is representative of the called behavior.